src/Fixer/ClassNotation/OrderedClassElementsFixer.php

```diff
@@ -217,7 +216,16 @@ public function C(){}
 ',
                     ['order' => ['method_public'], 'sort_algorithm' => self::SORT_ALPHA]
                 ),
-            ]
+            ],
+            'Accepts a subset of pre-defined element types, special element groups, and custom patterns.
+
+Element types: `[\''.implode('\', \'', array_keys(self::$typeHierarchy)).'\']`
+
+Special element types: `[\''.implode('\', \'', array_keys(self::$specialTypes)).'\']`
+
+Custom values:
+
+- `method:*`: specify a single method name (e.g. `method:__invoke`) to set the relative order of that specific method'
```

Comment:
I think that `relative` is misleading here. Relative to what? Maybe `explicit` would be better?

Reply:
> Relative to what?

Relative to the other class elements. Using something like `method:__invoke` doesn't set the explicit order of the `__invoke` method (e.g. 2nd or 3rd element in the class), it sets the order _relative to the other elements_ configured in this rule (e.g. _before_ or _after_ some other elements).

Not super attached to it though, I can just take that word out and I think it's still clear.

Reply:
Ok, so maybe we can use this instead?

```suggestion
- `method:*`: specify a single method name (e.g. `method:__invoke`) to set the explicit placement of that specific method'
```

Reply:
Sorry only just saw this comment now—I took out "relative", what do you think? Now it just says "to set the order of that specific method".

Reply:
It's OK with current description ("to set the order of that specific method"), so it's up to you what you like better - I'll just resolve this thread and you may apply suggestion as you wish 🙂.

Reply:
Thanks! I'm going to keep what's there now, just "set the order", but I appreciate the input. And of course feel free to change it yourself later if you think of something better!